pom.xml

```diff
@@ -784,6 +786,26 @@
 				<artifactId>javax.inject</artifactId>
 				<version>1</version>
 			</dependency>
+			<dependency>
+				<groupId>javax.xml.bind</groupId>
+				<artifactId>jaxb-api</artifactId>
+				<version>2.3.0</version>
+			</dependency>
+			<dependency>
+				<groupId>org.glassfish.jaxb</groupId>
+				<artifactId>jaxb-runtime</artifactId>
+				<version>2.3.0</version>
+			</dependency>
+			<dependency>
+				<groupId>javax.activation</groupId>
+				<artifactId>activation</artifactId>
+				<version>1.1.1</version>
+			</dependency>
+			<dependency>
+				<groupId>javax.xml.ws</groupId>
+				<artifactId>jaxws-api</artifactId>
+				<version>2.3.0</version>
```

Comment:
Shouldn't these all use a version variable?

Reply:
I don't know if it matters when the version is only referenced once. At least we seem to have quite a few other dependencies defined without a property variable. I think the properties are mostly useful for something like Spring or MetaModel or whatever, where the same version if referenced again and again.

Reply:
Fine with me, then. In general though, I'd prefer we either do it consistently or not at all. When it's there, you're kinda led to think it's the go-to place to look for versions. :)

In the long term, a complete parent POM dependencyManagment section along with some evil build-breaking checker for other places _might_ be better. That brings its own problems, though.